centreon/www/front_src/src/Dashboards/components/DashboardNavbar/DashboardNavbar.tsx

```diff
@@ -1,34 +1,8 @@
-import { useLocation } from 'react-router';
-import { useTranslation } from 'react-i18next';
-import { Link as RouterLink } from 'react-router-dom';
-import { includes } from 'ramda';
-
-import { Link } from '@mui/material';
-
-import { labelDashboards } from '../../translatedLabels';
-import routeMap from '../../../reactRoutes/routeMap';
 import FederatedComponent from '../../../components/FederatedComponents';
 
-import { useDashboardNavbarStyles } from './DashboardNavbar.styles';
-
 const DashboardNavbar = (): JSX.Element => {
-  const { classes } = useDashboardNavbarStyles();
-  const { t } = useTranslation();
-  const location = useLocation();
-
   return (
-    <nav className={classes.navbar}>
-      <Link
-        className={classes.link}
-        component={RouterLink}
-        data-selected={includes(routeMap.dashboards, location.pathname)}
-        to={routeMap.dashboards}
-        underline="hover"
-      >
-        {t(labelDashboards)}
-      </Link>
-      <FederatedComponent path="/it-edition-extensions/playlists/NavigationLink" />
-    </nav>
+    <FederatedComponent path="/it-edition-extensions/playlists/NavigationLink" />
```

Comment:
Why don't we want to show the link to dashboard?
Because It gonna have an impact somewhere it's displayed